Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs

```diff
@@ -17,15 +18,16 @@ public class WindowSizeEventArgsTests
         public void Ctor_WhenInvoked_SetsProperties()
         {
             // Arrange
-            var eventArgs = new WindowSizeEventArgs(123, 456);
+            (var heightArg, var widthArg) = (123, 456);
+            var eventArgs = new WindowSizeEventArgs(widthArg, heightArg);
 
             // Act
             var actualWidth = eventArgs.Width;
             var actualHeight = eventArgs.Height;
 
             // Assert
-            Assert.Equal(123, actualWidth);
-            Assert.Equal(456, actualHeight);
+            actualHeight.Should().Be(heightArg);
+            actualWidth.Should().Be(widthArg);
```

Comment:
This works just fine as it is right now. :)

Just a couple of things though.  For the purpose of keeping the code as simple and short as possible, I would just hard code the values into the `Be()` methods instead of creating a hard-coded tuple value that deconstructs into 2 separate variables.

**I suggest the example below:**
```cs
            // Arrange
            var eventArgs = new WindowSizeEventArgs(123, 456);

            // Act
            var actualWidth = eventArgs.Width;
            var actualHeight = eventArgs.Height;

            // Assert
            actualWidth.Should().Be(123);
            actualHeight.Should().Be(456);
```

Just a quick couple of tips.  Usually, when it comes to unit tests in general, it is better to be a little more verbose, have a little more layout such as the AAA setup like you see here, and have the code be as simple as possible.

The benefit of simple and easy-to-understand tests makes it a lot easier in the future to see why things are broken and what the expected behavior is.  Unit tests should act like documentation of your code in the sense of dependencies and behavior.